src/containers/Node/Node.tsx

```diff
@@ -50,11 +52,15 @@ export function Node(props: NodeProps) {
     );
     const loading = isFetching && currentData === undefined;
     const node = currentData;
+    const isDiskPagesAvailable = useDiskPagesAvailable();
```

Comment:
Please, use also `useCapabilitiesLoaded`. In case `viewer/capabilities` take some time to load, you have the chance to see structure tab